desktop/yast2-packager.desktop

```diff
@@ -2,7 +2,7 @@
 Encoding=UTF-8
 Name=Install/Remove Software
 GenericName=Install/Remove Software
-Exec=xdg-su -c "/sbin/yast2 sw_single %F"
```

Comment:
I think this means that the real bug is in /sbin/yast2 and we should fix that, instead of adding yet another script which is just overly complex quoting.